netkan/netkan/indexer.py

```diff
@@ -123,6 +125,8 @@ def status_attrs(self, new=False):
             # If we have perfomed an inflation, we certainly
             # have checked the mod!
             'last_checked': inflation_time,
+            # Copy the links to the status page
+            **(self._raw.get('resources', {})),
         }
```

Comment:
We should break out the CKAN metadata stuff from the mirroring PR, so we can consume it here.

Reply:
If you mean the `Ckan` class in `metadata.py`, that was part of the main webhooks PR and already available. I'll take a look at using it here...

Reply:
... done. I tested analogous expressions as much as I could in a `python3` prompt, so hopefully this won't throw exceptions.

Reply:
Now, do we want to breakout the metadata.py into a separate pr or merge the mirroring untested/unused for later inspection. The former would be easier for me to test + review, the later would get us moving on this faster.

Reply:
Like I said, `metadata.py` was already merged in #48 and has no changes in this PR.

Reply:
Oh gosh, I totally mis-read that and had made the assumption it wasn't available as you hadn't used it. Too many interruptions this morning while I was absorbing the first coffee of the day!